SF iOS/SF iOS/Helpers/Settings/NSUserDefaults+Settings.h

```diff
@@ -0,0 +1,20 @@
+//
+//  NSUserDefaults+Settings.h
+//  SF iOS
+//
+//  Created by Zachary Drayer on 4/8/19.
+//  Copyright © 2019 Amit Jain. All rights reserved.
```

Comment:
Is the project settings somehow adding his name to the file head?

Reply:
yup. to be honest, my eyes glaze over the comment header at the top of each file and i missed this

Reply:
pushed a commit to fix this + remove the copyright from files we both made

Reply:
Explicitly thanking you for removing it from mine too. 🙇 Saved me that anxiety when I noticed it 3 months from now.